C/CUDNN/common.jl

```diff
@@ -0,0 +1,57 @@
+include("../../fancy_toys.jl")
+
+version = v"7.6.5"
+
+name = "CUDNN_CUDA$(cuda_version.major).$(cuda_version.minor)"
```

Comment:
Should have noted before, but this is not a valid Julia identifier.  Same for CUTENSOR